ci/build_docs.sh

```diff
@@ -49,7 +49,7 @@ popd
 
 rapids-logger "Build Python docs"
 pushd docs/cudf
-make dirhtml
+make dirhtml O="-j 8"
```

Comment:
I don't know what `O` means here. I see a comment in the Makefile, `$(O) is meant as a shortcut for $(SPHINXOPTS).`. However, I think it's needlessly confusing to use a single character variable for this purpose. We should write `SPHINXOPTS` directly for clarity, or use a different name like `PARALLEL_LEVEL` and wire that through the Makefile.
```suggestion
make dirhtml SPHINXOPTS="-j 8 -n -v -W --keep-going"
```

Reply:
Bleh. I agree with you but the constraints here (trying to keep in line with the default Sphinx Makefile, not wanting to specify duplicate options, etc.) are kind of ugly. I'll accept your solution...

Reply:
I know, not my favorite either. We'll get around to the issue eventually...